diff --git a/src/appworld/environment.py b/src/appworld/environment.py
index 464a17e..1d4d8c1 100644
--- a/src/appworld/environment.py
+++ b/src/appworld/environment.py
@@ -568,11 +568,12 @@ class AppWorld:
                 f"This task ID {self.task_id} doesn't have a ground truth, "
                 "so it can't be evaluated."
             )
-        return appworld.evaluator.evaluate_task(
+        test_tracker, _ = appworld.evaluator.evaluate_task(
             task_id=self.task_id,
             experiment_name=self.experiment_name,
             suppress_errors=suppress_errors,
         )
+        return test_tracker
 
     def _save_api_calls_log(self) -> None:
         self._maybe_raise_remote_environment_error("_save_api_calls_log")
